zingolib/src/wallet/transaction_records_by_id/trait_inputsource.rs

```diff
@@ -160,6 +163,25 @@ impl InputSource for TransactionRecordsById {
             }
         }
 
+        if selected.len() < 2 {
+            // since we maxed out the target value with only one note, we have an option to grace a note.
+            // we will rescue the biggest dust note
+            unselected.reverse();
+            if let Some(smallest_note) = unselected
```

Comment:
This is mislabeled, and fixed in a different PR, please merge that one and resolve the conflict.